app/views/layouts/application.html.erb

```diff
@@ -85,7 +85,7 @@
 
               <% if current_admin.accessible_facilities(:manage_overdue_list).any? %>
                 <li class="nav-item">
-                  <%= link_to "Overdue patients", appointments_path, class: "nav-link #{active_controller?("appointments")}" %>
+                  <%= link_to "Overdue", appointments_path, class: "nav-link #{active_controller?("appointments")}" %>
```

Comment:
Why this change?

Reply:
accidentally included " patients" in an earlier PR, the old-permissions branch of the layout just says "Overdue" now, so I've removed the extra " patients" here.

Reply:
Noice 😎